modules/dxp/apps/analytics/analytics-reports-web/src/test/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProviderTest.java

```diff
@@ -172,6 +173,77 @@ public void testGetHistoricalReadsHistoricalMetric() throws Exception {
 		Assert.assertEquals(localDate, zonedDateTime.toLocalDate());
 	}
 
+	@Test
+	public void testGetReferringDomains() throws Exception {
+		AnalyticsReportsDataProvider analyticsReportsDataProvider =
+			new AnalyticsReportsDataProvider(
+				_getHttp(
+					Collections.singletonMap(
+						"/page-referrer-hosts",
+						JSONUtil.put(
+							"abc.com", 3.0
+						).put(
+							"google.com", 6.0
+						).put(
+							"liferay.com", 1.0
+						).toString())));
+
+		List<ReferringURL> referringURLS =
```

Comment:
Would it be `referringURLs ` better, there are some examples in portal

Reply:
Because Brian will not approve `referringDomains`, right?

Reply:
yeah!!